Tests/AppTests/AnalyzerTests.swift

```diff
@@ -70,7 +70,7 @@ class AnalyzerTests: AppTestCase {
         assert(commands: commands, expectations: expecations)
 
         // validate versions
-        // TODO: This is monstrous... create a helper? There has to be a better way?
+        // A bit awkward... create a helper? There has to be a better way?
         let pkg1 = try Package.query(on: app.db).filter(by: urls[0].url).with(\.$versions).first().wait()!
```

Comment:
"This is monstrous" -> "A bit awkward" with no code changes...

The next commit on this line is going to change it to "This is awesome!" 😂